### DISCUSSION TITLE
MoesGo ‘MS-105BZ’ in Pairing mode

### DISCUSSION THREAD

**Discussion by Infraviored** [Koenkk/zigbee2mqtt#17410] [Devices]
Dear Community,

i have a MS-105BZ dual zigbee dimmer.
I do not know how to get it into pairing mode.

In the manual, it says:

> After the wiring of switch module is done, press the reset switch for 10 times (keep time interval not too short) until the beep sounds continuously and rapidly as Di-Di-Di... for pairing mode.

I tried clicking the stupid buttons thousands of times at different intervals. There is just nothing changing. The lights turn on or off, but there is no pairing mode.

Does anybody have an idea?

**Comment by joostvanmourik** [comment]
Did you ever found a solution for this one i have the same problem

**Comment by oberlinatgithub** [comment]
For me it don't work too... 🙈
